src/sqlfluff/core/parser/segments/raw.py

```diff
@@ -56,6 +58,8 @@ def __init__(
         self._is_expandable = None
         # Keep track of any source fixes
         self._source_fixes = source_fixes
+        # UUID for matching
```

Comment:
Nitpick: Instead of this, should it call the class (`BaseSegment`) constructor instead?

Reply:
Yeah I thought about that. On balance I thought so much is different about the two constructors that inheriting seemed the more complex route.